ambry-server/src/integration-test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -982,11 +983,11 @@ protected static void endToEndReplicationWithMultiNodeSinglePartitionTest(String
       // read the replica file and check correctness
       // The token offset value of 13062 was derived as followed:
       // - Up to this point we have done 6 puts and 1 delete
-      // - Each put takes up 2177 bytes in the log (1000 data, 1000 user metadata, 177 ambry metadata)
+      // - Each put takes up 2179 bytes in the log (1000 data, 1000 user metadata, 179 ambry metadata)
       // - Each delete takes up 97 bytes in the log
       // - The offset stored in the token will be the position of the last entry in the log (the delete, in this case)
-      // - Thus, it will be at the end of the 6 puts: 6 * 2177 = 13062
-      checkReplicaTokens(clusterMap, dataNodeId, 13062, "0");
+      // - Thus, it will be at the end of the 6 puts: 6 * 2179 = 13074
+      checkReplicaTokens(clusterMap, dataNodeId, 13074, "0");
```

Comment:
Again, if you haven't, please run this in a loop on Linux and OS X, just to make sure.